pkg/cover/report_test.go

```diff
@@ -428,28 +428,20 @@ func checkCSVReport(t *testing.T, CSVReport []byte) {
 		t.Fatalf("no main in the CSV report")
 	}
 }
-func checkJSONReport(t *testing.T, r []byte) {
-	expected := []byte(`{
-	"version":1,
-	"total_cb_count":1,
-	"covered_cb_count":1,
-	"files": {
-		"main.c": {
-			"total_cb_count":1,
-			"covered_cb_count":1,
-			"functions":{
-				"main": {
-					"total_cb_count":1,
-					"covered_cb_count":1
-				}
-			}
-		}
-	}
-}`)
+
+// nolint:lll
+func checkJSONLReport(t *testing.T, r []byte) {
+	expected := []byte(`{"version":1,"file_path":"main.c","func_name":"main","sl":1,"sc":0,"el":1,"ec":` +
+		strconv.Itoa(backend.LineEnd) + `,"cb_hit_count":1,"inline":true,"pc":12345}`)
```

Comment:
This `backend.LineEnd` will end up in the database, but it was intended only as an internal const. Do we want something more abstract to denote infinity (0, -1, no value at all)?

Reply:
-1 looks better. Let's try -1.